tensorflow/python/kernel_tests/stage_op_test.py

```diff
@@ -102,8 +102,7 @@ def testColocation(self):
       with ops.device(gpu_dev):
         stager = data_flow_ops.StagingArea([dtypes.float32])
         y = stager.put([v])
-        self.assertEqual(y.device, '/device:GPU:0' if gpu_dev
-                                                   else gpu_dev)
+        self.assertEqual(y.device, gpu_dev)
```

Comment:
This cause the following test failure on gpu:
======================================================================
FAIL: testColocation (__main__.StageTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/tensorflow-pull-requests-gpu/bazel-ci_build-cache/.cache/bazel/_bazel_jenkins/eab0d61a99b6696edb3d2aff87b585e8/execroot/workspace/bazel-out/local_linux-opt/bin/tensorflow/python/kernel_tests/stage_op_test.runfiles/org_tensorflow/tensorflow/python/kernel_tests/stage_op_test.py", line 105, in testColocation
    self.assertEqual(y.device, gpu_dev)
AssertionError: u'/device:GPU:0' != u'/gpu:0'